docs/tools/vdb_table/data/aperturedb.json

```
@@ -127,33 +127,33 @@
     "comment": ""
   },
   "ephemeral": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
-    "comment": ""
+    "comment": "Not documented externally."
   },
   "sharding": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
-    "comment": ""
+    "comment": "Not documented externally."
   },
   "doc_size": {
     "bytes": 0,
-    "unlimited": false,
-    "source_url": "",
-    "comment": ""
+    "unlimited": true,
+    "source_url": "https://docs.aperturedata.io/administration/Configuring%20ApertureDB",
+    "comment": "You can store large or small files but need to change message size for adding or querying files larger than 256MB. For very large files like GBs of video, it will require optimizations to send over network or would be slow."
```

I think it would be easier for the reader if they knew the exact parameter to look for in the docs - minor suggestion here.
```suggestion
    "comment": "You can store large or small files but need to change the `max_connection_message_size_mb` for adding or querying files larger than 256MB. For very large files like GBs of video, it will require optimizations to send over network or would be slow."
```